### DISCUSSION TITLE
Debugger requirements in the IBM i server

### DISCUSSION THREAD

**Discussion by ToshBimbra** [codefori/vscode-ibmi#1613] [General]
VSC IBM i Debugger requires 
**5770WDS   60       5050    Workstation Tools - Base** 
Please update the documentation to reflect this requirement.

**Comment by sebjulliand** [comment]
Hi @ToshBimbra , thanks for the heads up.
If you want, you can fork the documentation repository and submit a PR with the change you'd like to see: https://github.com/codefori/docs
Opening an issue on the documentation repository is fine too if you don't feel like making the change yourself.
